TypeCobol.Test/Parser/Programs/Cobol85/ProgramEndMismatchStacked0.rdz.cbl

```diff
@@ -0,0 +1,19 @@
+﻿       IDENTIFICATION DIVISION.
+       PROGRAM-ID. Stacked0.
```

Comment:
Not much but I'd rather have the Main program named Main (also for other tests) and the test renamed `ProgramEndMismatchMain` in that case as to be easier when looking for a specific unit test and for reading :).

Reply:
Tests have been updated as you suggested.